data/colors/themes/dark_high_contrast.ts

```diff
@@ -198,6 +198,18 @@ const exceptions = {
       numBg: alpha(get('accent.muted'), 0.4),
     },
   },
+  
+  actionListState: {
+    default: {
+      hoverBg: 'rgba(201, 206, 212, 0.12)',
+      focusBg: 'rgba(201, 206, 212, 0.24)'
+    },
+    danger: {
+      hoverBg: get('danger.muted'),
+      hoverText: get('fg.onEmphasis'),
```

Comment:
@auareyou @Juliusschaeper do we have ways to validate how this looks in high contrast? Here's what I'm seeing this when testing these values: ☹️

![Screen Shot 2021-10-08 at 16 20 21](https://user-images.githubusercontent.com/293280/136634038-21db9aa2-dc42-4433-bf7c-18f1c5790499.png)

Also, what's your recommended approach for interactive states in high contrast? Do the `:hover`, `:focus` styles become solid? As opposed to some of our current components (that show a blue background with white text), `actionList` doesn't invert the text color since it can support a lot of rich content inside. But I'm not sure if that's something we should do in high contrast.

Reply:
@vdepizzol and me met to discuss HC states for the action list:

- For the default item we focused on having a clearly differentiable outline while keeping the fill as it is to allow for rich content. 
- The danger item follows the general HC approach of using emphasis + fg.onEmphasis.
- _Now looking at it, I wonder if the .selected state should also use emphasis + fg.onEmphasis 🤔_

<img width="1457" alt="Dark HC Neutral" src="https://user-images.githubusercontent.com/2427008/138283363-82f8231d-15bd-4ea0-8490-797e859ce975.png">
<img width="1457" alt="Dark HC Danger" src="https://user-images.githubusercontent.com/2427008/138283400-2ae37c2c-31c7-4af5-9b64-53ce9c9523b5.png">